templates/configuration/graphdb-users-configmap.yaml

```diff
@@ -0,0 +1,15 @@
+# Default configuration map for provisioning GraphDB repository.
```

Comment:
same as above

Reply:
Fixed for all 5 configmaps that contain the config files.